cmd/lansible

```diff
@@ -314,9 +314,16 @@ EOF
   elif [[ "$@" == *--playbook* ]] || [[ "$@" == *--with-playbook* ]]; then
     strap::bot "Run Ansible Playbook"
     strap::ansible::playbook::run "$@"
-  elif [[ -d "${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default" ]]; then
-    strap::bot "Run Ansible Playbook"
-    strap::ansible::playbook::run --with-playbook="${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default" "$@" 
+  elif [[ -z "$@" ]]; then
+    if [[ -f "${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default/main.yml" ]]; then
+      strap::bot "Run Ansible Playbook"
+      strap::ansible::playbook::run --with-playbook="${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default/main.yml" "$@"
+    else
+      printf "\nDefault playbook ${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default/main.yml not found. No ansible playbook will be run."
```

Comment:
Let's not use printf - there should be a strap error output function for this to ensure that these messages are directed to stderr

Reply:
Sure.